src/status_im2/subs/pairing.cljs

```diff
@@ -19,4 +19,8 @@
 (re-frame/reg-sub
  :pairing/installation-name
  :<- [:multiaccount]
- (fn [multiaccount] (:installation-name multiaccount)))
\ No newline at end of file
+ (fn [multiaccount] (:installation-name multiaccount)))
+
+(re-frame/reg-sub
+ :pairing/pairing-in-progress
+ (fn [pairing-in-progress?] (:syncing pairing-in-progress?)))
```

Comment:
I believe you receiving the whole `app-db` as the function argument and as the guidelines suggest to have [layer 3 subscriptions](https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#registering-layer-3-subscriptions), the `:pairing/pairing-in-progress` subscription needs to be converted as below:

```clojure

;; status-im2.subs.root
(reg-root-key-sub :syncing :syncing)

;; status-im2.subs.pairing
(re-frame/reg-sub
 :pairing/pairing-in-progress
 :<- [:syncing]
 (fn [syncing] 
    (:pairing-in-progress? syncing)))
```